core/sqf/monitor/linux/process.cxx

```diff
@@ -1317,7 +1318,11 @@ bool CProcess::Create (CProcess *parent, int & result)
     env = getenv ("TERM");
     STRCPY (term, (env?env:"ansi"));
     env = getenv ("TZ");
-    STRCPY (tz, (env?env:""));
+    tz_exists = (env != NULL);
+    if (tz_exists)
+    {
+      STRCPY (tz, env); // see note regarding TZ below
+    }
     env = getenv ("USER");
     STRCPY (user, (env?env:""));
     env = getenv ("HOME");
```

Comment:
Thanks Dave for this find! Requesting @zcorrea also to take a look at the process.cxx changes.